src/types.rs

```diff
@@ -482,6 +482,30 @@ impl Rewrite for ast::Ty {
                 let budget = try_opt!(width.checked_sub(2));
                 ty.rewrite(context, budget, offset + 1).map(|ty_str| format!("({})", ty_str))
             }
+            ast::TyTup(ref tup_ret) => {
+                let inner = tup_ret.iter()
+                                   .map(|item| item.rewrite(context, width, offset))
+                                   .fold(Some("".to_owned()),
+                                         |sum, x| {
+                                             match (sum, x) {
+                                                 (Some(sum), Some(x)) => {
+                                                     if sum == "" {
+                                                         // First item.
+                                                         Some(x)
+                                                     } else {
+                                                         Some(sum + ", " + &x)
+                                                     }
+                                                 }
+                                                 _ => None,
+                                             }
+                                         });
+
+                if let Some(inner) = inner {
```

Comment:
Nit: we have the `try_opt!` macro for this. Could you call `wrap_str` on the resulting string? That way we make sure it fits within the given width.